### PR TITLE
Fix hardhat log

### PR DESCRIPTION
**Pull request by yann300**
fix https://github.com/ethereum/remix-project/issues/1712

manual test:
 - open the default tests file.
 - add "import "hardhat.sol".
 - put a console.log in the last function.
 - deploy the test, and execute function in order.
 - the last function should print the log.

### CODE REVIEW

**Comment by Aniket-Engg** [comment]
After loading the remix IDE from IPFS published link, it shows:
```
retainable.js:20 Unhandled promise rejection TypeError: Object is not iterable.
    at i (main.js:2)
    at main.js:2
    at main.js:2
    at Object.next (main.js:2)
    at main.js:2
    at new T (retainable.js:20)
    at o (main.js:2)
    at x (main.js:2)
    at B.<anonymous> (main.js:2)
    at main.js:2
```

After some time, it shows this error everytime I try to update a file in editor:

```
Uncaught Error: ((intermediate value)(intermediate value)(intermediate value) || []) is not iterable

TypeError: ((intermediate value)(intermediate value)(intermediate value) || []) is not iterable
    at Mt.maybeShowAt (editor.main.js:744)
    at ci._onEditorMouseMove (editor.main.js:744)
    at editor.main.js:744
    at C.fire (editor.main.js:60)
    at N.ce.onMouseMove (editor.main.js:636)
    at N.emitMouseMove (editor.main.js:593)
    at L.emitMouseMove (editor.main.js:634)
    at s._onMouseMove (editor.main.js:593)
    at editor.main.js:593
    at fe (editor.main.js:66)
    at editor.main.js:6
```

Not sure if this is related to this PR.

**Review by Aniket-Engg — COMMENTED** [review]
Couldn't test logging as accounts were not there. Accounts don't appear even after refreshing or on fresh load.
Screenshot 2021-11-03 at 12 06 57 PM

**Comment by yann300** [comment]
@Aniket-Engg could you test it with a local build?

**Comment by Aniket-Engg** [comment]
> @Aniket-Engg could you test it with a local build?

Those errors are not there locally and logging works fine but for `[call]`, tx status is still `undefined`

File: package.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     "nightwatch_local_staticAnalysis": "npm run build:e2e && nightwatch --config dist/apps/remix-ide-e2e/nightwatch.js dist/apps/remix-ide-e2e/src/tests/staticAnalysis.spec.js --env=chrome",
     "nightwatch_local_signingMessage": "npm run build:e2e && nightwatch --config dist/apps/remix-ide-e2e/nightwatch.js dist/apps/remix-ide-e2e/src/tests/signingMessage.test.js --env=chrome",
     "nightwatch_local_specialFunctions": "npm run build:e2e && nightwatch --config dist/apps/remix-ide-e2e/nightwatch.js dist/apps/remix-ide-e2e/src/tests/specialFunctions.test.js --env=chrome",
-    "nightwatch_local_solidityUnitTests": "npm run build:e2e && nightwatch --config dist/apps/remix-ide-e2e/nightwatch.js dist/apps/remix-ide-e2e/src/tests/solidityUnittests.spec.js --env=chrome",
+    "nightwatch_local_solidityUnitTests": "npm run build:e2e && nightwatch --config dist/apps/remix-ide-e2e/nightwatch.js dist/apps/remix-ide-e2e/src/tests/solidityUnittests.spec.js --env=firefox",
```
Comment (ioedeveloper) [inline package.json:75]: I don't think this change is needed. `'chrome' => 'firefox'`

**Review by ioedeveloper — APPROVED** [review]

**Pull request merged**